test/any/confluence/lone_omission-test.es6.js

```diff
@@ -198,72 +204,64 @@ describe('LoneOmission', function() {
         mkData(0, date1, beta1, [], [alpha1]),
         mkData(0, date3, beta2, [beta1], [alpha2]),
       ])).toBe(true);
-      done();
-    });
+    }).then(done, done.fail);
   });
 
   it('should exclude APIs not shipped by even one compared release', function(done) {
-    let alpha2, alpha2_1, beta2, beta2_1, charlie2, charlie2_1;
     // Use date2 and date2_1 to ensure two versions during grace period.
-    Promise.all([
-      releases.put(Release.create({
+    let [alpha2, alpha2_1, beta2, beta2_1, charlie2, charlie2_1] = init([
+      {
         browserName: 'Alpha',
         browserVersion: '2',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2,
-      }, container)),
-      releases.put(Release.create({
+      },
+      {
         browserName: 'Alpha',
         browserVersion: '2.1',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2_1,
-      }, container)),
-      releases.put(Release.create({
+      },
+      {
         browserName: 'Beta',
         browserVersion: '2',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2,
-      }, container)),
-      releases.put(Release.create({
+      },
+      {
         browserName: 'Beta',
         browserVersion: '2.1',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2_1,
-      }, container)),
-      releases.put(Release.create({
+      },
+      {
         browserName: 'Charlie',
         browserVersion: '2',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2,
-      }, container)),
-      releases.put(Release.create({
+      },
+      {
         browserName: 'Charlie',
         browserVersion: '2.1',
         osName: 'Windows',
         osVersion: '10',
         releaseDate: date2_1,
-      }, container)),
-    ]).then(function(releasesArray) {
-      alpha2 = releasesArray[0];
-      alpha2_1 = releasesArray[1];
-      beta2 = releasesArray[2];
-      beta2_1 = releasesArray[3];
-      charlie2 = releasesArray[4];
-      charlie2_1 = releasesArray[5];
-
-      const iface = mkIface('NotIn', 'charlie2');
-      return Promise.all([
-        ifaces.put(iface),
-        junctions.put(mkJunction(beta2, iface)),
-        junctions.put(mkJunction(beta2_1, iface)),
-        junctions.put(mkJunction(charlie2_1, iface)),
-      ]);
-    }).then(function() {
+      },
+    ]);
+    
+    Promise.all([
+      releases.put(alpha2),
+      releases.put(alpha2_1),
+      releases.put(beta2),
+      releases.put(beta2_1),
+      releases.put(charlie2),
+      releases.put(charlie2_1),
```

Comment:
I think this test is missing the interface setup.

Reply:
Done.